include/qubes-gui-protocol.h

```diff
@@ -55,6 +55,11 @@
  *  - gui-daemon sends MSG_WINDOW_DUMP_ACK to acknowledge finishing
  *    MSG_WINDOW_DUMP processing; this allows agent to know when it
  *    can safely unmap a window's grants
+ * 1.8:
+ *  - agent could send up to 256KB of clipboard data. And if clipboard is over
+ *    the limit, it will send one byte over it to trigger rejection on xside.
```

Comment:
Can you clarify that sending 256KB+1 should not result in connection termination anymore? The current "rejection on xside" is `exit(1)`...

Reply:
> The current "rejection on xside" is `exit(1)`...

I deleted that exit(1). It is red in color. [Line 1025](https://github.com/QubesOS/qubes-gui-daemon/pull/150/files#diff-7897f6116dd3815ad17499205f816d770805aca6ce63e96ff55be4fe2d95a3d6L1025)